lms/static/js/revisions/collections/revision_collection.js

```diff
@@ -0,0 +1,48 @@
+;(function (define) {
+
+define([
+    'backbone',
+    'js/revisions/models/revision'
+], function (Backbone, Revision) {
+    'use strict';
+
+    return Backbone.Collection.extend({
+
+        model: Revision,
+        url: '/api/revisions/',
+        fetchXhr: null,
+
+        initialize: function (models, options) {
+            Backbone.Collection.prototype.initialize.apply(this, arguments);
+        },
+
+        fetchRevisions: function () {
+            this.fetchXhr && this.fetchXhr.abort();
+            this.resetState();
+            this.fetchXhr = this.fetch({
+                data: {},
+                type: 'GET',
+                success: function (self, xhr) {
+                    self.trigger('revisions_loaded');
+                },
+                error: function (self, xhr) {
+                    self.trigger('error');
+                }
+            });
+        },
+
+        parse: function(response) {
+            return response;
+        },
```

Comment:
@bdero I forgot to ask last time: Is there a specific reason for overriding Backbone's default `Collection.parse` method here? Since this implementation returns the original response without modifying it, I'm wondering if we could remove it.